jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/kafka/listeners/Asn1DecodeMAPJSONListener.java

```diff
@@ -0,0 +1,80 @@
+package us.dot.its.jpo.ode.kafka.listeners;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.tomcat.util.buf.HexUtils;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.kafka.annotation.KafkaHandler;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Component;
+import us.dot.its.jpo.ode.kafka.OdeKafkaProperties;
+import us.dot.its.jpo.ode.model.Asn1Encoding;
+import us.dot.its.jpo.ode.model.OdeAsn1Data;
+import us.dot.its.jpo.ode.model.OdeAsn1Payload;
+import us.dot.its.jpo.ode.model.OdeMapMetadata;
+import us.dot.its.jpo.ode.model.OdeObject;
+import us.dot.its.jpo.ode.uper.StartFlagNotFoundException;
+import us.dot.its.jpo.ode.uper.SupportedMessageType;
+import us.dot.its.jpo.ode.uper.UperUtil;
+
+import java.util.Set;
+
+@Slf4j
+@Component
+@KafkaListener(id = "Asn1DecodeMAPJSONListener", topics = "${ode.kafka.topics.raw-encoded-json.map}")
+public class Asn1DecodeMAPJSONListener {
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    private final String publishTopic;
+    private final KafkaTemplate<String, OdeObject> kafkaTemplate;
+    private final Set<String> disabledTopics;
+
+    public Asn1DecodeMAPJSONListener(KafkaTemplate<String, OdeObject> kafkaTemplate,
+                                     @Value("${ode.kafka.topics.asn1.decoder-input}") String publishTopic,
```

Comment:
note: this is the future of how we'll reference the topics. once the full migration is complete, I'll be deleting all of the `*Topics.java` config objects in favor of explicit config references for better visibility of where each topic is used